.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+  },
+};
```

Comment:
We need to use [this](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) version instead.